pkg/pillar/cmd/volumemgr/updatestatus.go

```diff
@@ -595,7 +595,7 @@ func doUpdateVol(ctx *volumemgrContext, status *types.VolumeStatus) (bool, bool)
 		}
 	}
 	if status.State == types.CREATING_VOLUME && status.SubState == types.VolumeSubStatePreparing {
-		if ctx.persistType == types.PersistZFS && !status.IsContainer() {
+		if ctx.useVHost && ctx.persistType == types.PersistZFS && !status.IsContainer() {
```

Comment:
Maybe it's better to not to let the installation to proceed with the combination of `fs==zfs` and `hyper!=kvm`?

Reply:
makes sense. You already have my LGTM